### ISSUE TITLE
GPaste extension causes temporary Wayland lockup

### ISSUE THREAD

**Issue by jdoss** [Keruspe/GPaste#247]
Over time the GPaste Gnome Shell extension causes gnome-shell to stop responding a when highlighting text. 

```
Jul 02 12:57:29 sts71 org.gnome.Shell.desktop[2892]: == Stack trace for context 0x55daf24981f0 ==
Jul 02 12:57:29 sts71 org.gnome.Shell.desktop[2892]: #0 0x7ffca8412440 b   /usr/share/gnome-shell/extensions/GPaste@gnome-shell-extensions.gnome.org/indicator.js:292 (0x7f94d32db5e8 @ 37)
Jul 02 12:57:29 sts71 org.gnome.Shell.desktop[2892]: #1 0x7ffca84124e0 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f95100b5de0 @ 71)
Jul 02 12:57:29 sts71 org.gnome.Shell.desktop[2892]: #2 0x7ffca84125a0 b   /usr/share/gnome-shell/extensions/GPaste@gnome-shell-extensions.gnome.org/indicator.js:284 (0x7f94d32db450 @ 263)
Jul 02 12:57:29 sts71 gnome-shell[2892]: Object St.BoxLayout (0x55daf6ff5790), has been already deallocated - impossible to access to it. This might be caused by the fact that the object has been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
```

If you give it some time, your gnome-shell session becomes responsive again and you can see these events happening in the journal.

```
$ gnome-shell --version
GNOME Shell 3.28.2
$ rpm -qa |grep gpaste
gpaste-libs-3.28.2-1.fc28.x86_64
gpaste-ui-3.28.2-1.fc28.x86_64
gnome-shell-extension-gpaste-3.28.2-1.fc28.noarch
gpaste-3.28.2-1.fc28.x86_64
```

**Comment by lkraav** [comment]
Yes, I've repeatedly experienced text selection in `electron` apps like Slack completely freeze my desktop environment (Gnome 3.26). This only happens on Wayland, so I'm forced to stay w/ X for now.

**Comment by Keruspe** [comment]
This has been fixed in mutter 3.38
